defs/params.go

```diff
@@ -5,27 +5,30 @@ import (
 )
 
 var (
-	// InputLogMaxMessageBytes defines the maximum length of a log message, if such a field exists (parser dependent)
+	// InputLogMaxTotalBytes defines the maximum length of a log from input.
+	InputLogMaxTotalBytes = InputLogMaxMessageBytes + 256
+
+	// InputLogMaxMessageBytes defines the maximum length of a log message, if such a field exists (parser dependent).
 	//
-	// If the limit is exceeded, the message should be truncated and may be recorded in metrics
+	// If the limit is exceeded, the message is truncated and recorded in metrics.
 	InputLogMaxMessageBytes = 1 * 1024 * 1024
 
-	// InputLogMinMessageBytesToPool defines the minimum length of a log message to start using object pooling
+	// InputLogMinTotalBytesToPool defines the minimum length of a log to start using object pooling.
```

Comment:
`Min` and `Total` together sound kinda confusing, but that's not much of an issue I guess

Reply:
renamed to "Record"